src/Congo.RazorPages/Services/IProductsService.cs

```diff
@@ -7,7 +8,6 @@ namespace Congo.RazorPages.Services
     public interface IProductsService
     {
         Task<IEnumerable<Product>> GetProducts();
-        IEnumerable<Product> GetSampleProducts();
-        void PurchaseProduct(int id);
+        Task<Guid> Purchase(Guid productId);
```

Comment:
```suggestion
        Task<Guid> PurchaseAsync(Guid productId);
```

Adding async helps the devs know that they must await it